templates/chat/turn-card.hbs

```diff
@@ -18,9 +18,27 @@
     {{/if}}
 
     {{!-- Actions --}}
-    <!-- TODO: Add actions when implementing https://github.com/foundryvtt/dnd5e/issues/4861 -->
+    {{#if activities.length}}
+    <section class="activities">
+        <strong class="roboto-condensed-upper">{{ localize "DND5E.CHATMESSAGE.TURN.Activities" }}</strong>
+        <ul class="unlist">
+            {{#each activities}}
+            <li class="activity flexrow item-tooltip" data-activity-uuid="{{ uuid }}" data-item-uuid="{{ item.uuid }}">
+                <img class="gold-icon" src="{{ item.img }}" alt="{{ item.name }}">
+                <span class="name-stacked">
+                    <span class="title">{{ item.name }}</span>
+                    <span class="subtitle">{{ name }}</span>
+                </span>
+                <button type="button" data-action="use">
+                    {{ localize "DND5E.Use" }}
+                </button>
```

Comment:
This button looks a bit chunky in-line compared to our other in-line button style in the case of applying effects from the effects tray. I was thinking of using a single icon and making it smaller to match the effects tray icon, but an icon to represent 'use' might be a bit abstract.

Could we instead remove this button and give the item header the `rollable` class instead? Clicking an item header is pretty well established in the system by now as a way to 'use' it I think. That also frees up the right-hand side in case we want to put something else there (maybe charges or something).